### PR TITLE
Feed parsing: use author email when there's no author name

### PR DESCRIPTION
**Pull request by perrinjerome**
This is especially useful because when author is given as
`<author>Author Name</author>` ( as in this example
https://cyber.harvard.edu/rss/rss.html#ltauthorgtSubelementOfLtitemgt ),
SimplePie will expose *Author Name* as `email`.

### CODE REVIEW

**Comment by perrinjerome** [comment]
A random feed where authors were not parsed properly:  https://devnet.kentico.com/rss/articles 

I could not find which test could be extended to cover this, so I did not write any new test.

File: app/Models/Feed.php [hunk 1]
```diff
@@ -355,7 +355,7 @@ public function loadEntries($feed) {
 				$this->id(),
 				$item->get_id(false, false),
 				$title === null ? '' : $title,
-				$author === null ? '' : html_only_entity_decode(strip_tags($author->name)),
+				$author === null ? '' : html_only_entity_decode(strip_tags($author->name === null ? $author->email : $author->name)),
```
Comment (Alkarex) [inline app/Models/Feed.php]: I would suggest `== ''` instead of `=== null` to catch other nullish values such as empty strings

**Comment by Alkarex** [comment]
Thanks! 👍

File: app/Models/Feed.php (hunk 1 above)
Comment (perrinjerome) [reply app/Models/Feed.php]: Good point, I pushed-force with a fix for that

**Pull request merged**